docs/dev/setup/jenkins-localvc.rst

```diff
@@ -0,0 +1,797 @@
+    chosen username and password:
+
+    .. code:: sql
+
+        INSERT INTO `artemis`.`jhi_user` (`id`,`login`,`password_hash`,`first_name`,`last_name`,`email`,
+        `activated`,`lang_key`,`activation_key`,`reset_key`,`created_by`,`created_date`,`reset_date`,
+        `last_modified_by`,`last_modified_date`,`image_url`,`last_notification_read`,`registration_number`)
+        VALUES (1,"artemis_admin","HASHED_PASSWORD","artemis","administrator","artemis_admin@localhost",
+        1,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL);
+3. Give the user admin and user roles:
+
+    .. code:: sql
+
+        INSERT INTO `artemis`.`jhi_user_authority` (`user_id`, `authority_name`) VALUES (1,"ROLE_ADMIN");
+        INSERT INTO `artemis`.`jhi_user_authority` (`user_id`, `authority_name`) VALUES (1,"ROLE_USER");
+
+Starting the Artemis server should now succeed.
+
+Jenkins
+^^^^^^^
+
+Automated Jenkins Server Setup
+""""""""""""""""""""""""""""""
+
+The following steps describe how to deploy a pre-configured version of the Jenkins server.
+This is ideal as a quickstart for developers. For a more detailed setup, see
+`Manual Jenkins Server Setup <#manual-jenkins-server-setup>`__.
+In a production setup, you have to at least change the user credentials (in the file ``jenkins-casc-config-localvc.yml``) and push tokens.
+
+1. You can now first build and deploy Jenkins, then you can also start the other services which weren't started yet:
+
+    .. code:: bash
+
+       JAVA_OPTS=-Djenkins.install.runSetupWizard=false docker compose -f docker/<Jenkins setup to be launched>.yml up --build -d jenkins
+       docker compose -f docker/<Jenkins setup to be launched>.yml up -d
+
+   Jenkins is then reachable under ``http://localhost:8082/`` and you can login using the credentials specified
+   in ``jenkins-casc-config-localvc.yml`` (defaults to ``artemis_admin`` as both username and password).
+
+2. The `application-local.yml` must be adapted with the values configured in ``jenkins-casc-config-localvc.yml``:
+
+.. code:: yaml
+
+    artemis:
+        user-management:
+            use-external: false
+            internal-admin:
+                username: artemis_admin
+                password: artemis_admin
+        version-control:
+            url: http://172.17.0.1:8080 # `http://host.docker.internal:8080` for Windows
+            user: root
+            password: dummy # have to be set, but does not matter for LocalVC
+            build-agent-git-username: jenkins
+            build-agent-git-password: artemis_admin # choose some strong password and username (gives read access to all repositories)
+        continuous-integration:
+            user: artemis_admin
+            password: artemis_admin
+            url: http://localhost:8082
+            vcs-credentials: artemis_localvc_credentials
+            artemis-authentication-token-key: artemis_notification_plugin_token
+            artemis-authentication-token-value: artemis_admin
```

Comment:
Should be indented once to be part of item 2 and make the automatic numbering work.

Reply:
Addressed with https://github.com/ls1intum/Artemis/commit/d8ca39e99d9de1ded6737ebe81423a73916b236b